pkg/apis/pipeline/v1/param_types.go

```diff
@@ -414,6 +407,11 @@ type ParamValue struct {
 	ObjectVal map[string]string
 }
 
+// ArrayOrString is deprecated, this is to keep backward compatibility
+//
+// Deprecated: Use ParamValue instead.
+type ArrayOrString = ParamValue
+
```

Comment:
This is already deprecated; there's no reason to introduce it into the v1 apis package.

Reply:
thanks, updated.